src/main/java/org/gridsuite/modification/server/NetworkModificationController.java

```diff
@@ -77,4 +77,41 @@
     public ResponseEntity<Flux<UUID>> getModificationGroups() {
         return ResponseEntity.ok().body(networkModificationService.getModificationGroups());
     }
+
+    @PutMapping(value = "/networks/{networkUuid}/lines/{lineId}/lockout")
```

Comment:
This is a not a classical way for designing a REST API. REST API are more about state representation, not actions (unlike RPC). Why not something like:
PUT /networks/{networkUuid}/lines/{lineId}/status with the status value in the body?